commands-v2/ramsete/constants.py

```diff
@@ -0,0 +1,75 @@
+from commands2 import RamseteCommand
+
+from wpilib import AnalogGyro
+
+from wpimath.kinematics import DifferentialDriveKinematics
+
+from math import pi
+
+"""
+A place for the constant values in the code
+that may be used in more than one place. 
+"""
+
+
+class Constants:
+    def __init__(self):
+
+        # Universal gyro.
+        self.gyroObject = AnalogGyro(1)
```

Comment:
This isn't a constant?

Reply:
I was trying to create a global gyro for both the physics and drivetrain. Is it possible to create two objects with the same ID?